payloads/library/GitBunnyGit/payload.txt

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+
+# Title:     Git Bunny Git
+# Author:    Draxiom && audibleblink
+# Target:    any
+# Version:   1.0
+#
+# Update payloads from Github
+#
+# White            |  Ready (to run bb.sh on the host computer)
+# Red              |  No Internet
+# Red (blinking)   |  Could not mount filesystem
+# Amber            |  Running
+# Green (blinking) |  Git Pull Finished
+# Green            |  Git Clone Finished
+
+LED R G B
+ATTACKMODE RNDIS_ETHERNET 
+#ATTACKMODE ECM_ETHERNET
+
+# Set your desired repo url and branch if you're not looking to update from mainline
+git_repo="https://github.com/hak5/bashbunny-payloads.git"
+git_branch="master"
+payloads_dir="/root/udisk"
+touch /tmp/git
```

Comment:
I had that in there to try and be able to run `tail -f /tmp/git` as early as possible... but it still doesn't seem to pop up until the amber light goes on. I'll remove it.